### DISCUSSION TITLE
Is example code (test folder) missing?

### DISCUSSION THREAD

**Discussion by Tahutipai** [Arksine/moonraker#386] [Q&A] [answered]
In the [web api docs](https://github.com/Arksine/moonraker/blob/master/docs/web_api.md) it says:

> The test/client folder includes a basic test interface with example usage

Yet I can not see a `test` or `client` folder in the repository (although there is a `tests` folder for use with pytest I belive.
 
Am I missing something, or was there once a test folder with example code that has since been removed?

Goal: just looking to experiement with building a simple client to interface with Moonraker.

**Comment by Arksine** [comment, marked as answer]
Oops, stale documentation, I moved the test client to its own repo.  I'll fix that, In the mean time you can find it [here](http://github.com/arksine/moontest).

**Reply by Tahutipai** [reply]
Thankyou!
